tests/trout/models.py

```diff
@@ -73,9 +73,12 @@ class Comment(models.Model):
     )
 
     post_many_to_many = models.ManyToManyField(Post, through=PostToComment)
-    post_many_to_many_nullable = models.ManyToManyField(
-        Post, through=PostToComment, null=True
-    )
+    # NOTE: null has no meaning for ManyToMany and django just ignores it and warns about it
+    # post_many_to_many_nullable = models.ManyToManyField(
+    #     Post,
+    #     through=PostToComment,
+    #     null=True,
+    # )
```

Comment:
I didn't remove this, but null=True has absolutely no effect here: https://github.com/django/django/blob/main/django/db/models/fields/related.py#L1237